### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by Levils114** [formidable-webview/ersatz#6]
I'm trying to run doc example, and getting this error:

```shell
    Test suite failed to run
    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • If you are trying to use ECMAScript Modules, see https://jestjs.io/docs/en/ecmascript-modules for how to enable it.
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /Users/{my_project_name}/node_modules/@formidable-webview/ersatz/node_modules/node-fetch/src/index.js:9
    import http from 'node:http';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime.createScriptFromCode (node_modules/jest-runtime/build/index.js:1350:14)
      at Object.<anonymous> (node_modules/@formidable-webview/ersatz/lib/commonjs/SourceLoader.tsx:3:1)
```

I tried to put ```'@babel/plugin-transform-react-jsx'``` in ```babel.config.js```, but this don't resolve the problem.

Code in ```config/__mocks__/react-native-webview.js```:
```js
import Ersatz from '@formidable-webview/ersatz';
export default Ersatz;
```

Code in  ```src/components/MyComponent/tests/MyComponent.test.tsx```:
```tsx
jest.mock('react-native-webview');
import * as React from 'react';
import { waitForWindow } from '@formidable-webview/ersatz-testing';
import { render } from '@testing-library/react-native';
import { default as WebView, WebViewProps } from 'react-native-webview';
 
const MyComponent = ({ source }: Pick<WebViewProps, 'source'>) => (
  <WebView source={source} injectedJavaScript={'window.awesomeGlobal = 1;'} />
);
 
describe('MyComponent', () => {
  it('should make awesomeGlobal available to window with value “1”', async () => {
    const window = await waitForWindow(
      render(<MyComponent source={{ html: '<div></div>' }} />)
    );
    expect(window.awesomeGlobal).toEqual(1);
  });
});
```

**Comment by jsamr** [comment]
@Levils114 See this comment https://github.com/formidable-webview/ubiquitous/issues/5#issuecomment-953703830

**Comment by Levils114** [comment]
I tried change ```babel.config.js``` to this:

```js
module.exports = {
  presets: ['module:metro-react-native-babel-preset'],
  plugins: ["inline-dotenv", '@babel/plugin-transform-react-jsx'] 
};
```

but the error persists.

**Comment by jsamr** [comment]
@Levils114 I am providing in a fix. In the meantime, if you use `yarn` packager, you can add this to your package.json:

```json
  "resolutions": {
    "node-fetch": "3.0.0-beta.9"
  },
```

**Comment by Levils114** [comment]
> @Levils114 I am providing in a fix. In the meantime, if you use `yarn` packager, you can add this to your package.json:
> 
> ```json
>   "resolutions": {
>     "node-fetch": "3.0.0-beta.9"
>   },
> ```

Thank you, this resolve the problem :D

**Comment by jsamr** [comment]
@Levils114 In any case, it should be fixed in just-released version 2.1.3!
